src/index.scss

```diff
@@ -0,0 +1,129 @@
+$night0: hsl(137, 8%, 17%);
+$night1: hsl(137, 5%, 88%);
+
+$grey0: hsl(0, 0%, 98%);
+$grey1: hsl(175, 5%, 75%);
+
+$purple: hsl(240, 100%, 76%);
+$purple2: hsl(240, 100%, 66%);
+
+$aqua0: hsl(175, 100%, 20%);
+$aqua1: hsl(175, 100%, 25%);
+$aqua2: hsl(175, 100%, 15%);
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: $night0;
+}
+
+.flexCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.loginContainer {
+  @extend .flexCenter;
+  flex-direction: column;
+  border-radius: 7px;
+  height: 400px;
+  width: 520px;
+  margin: 13% auto 0;
+  background: $night1;
+
+  &__registerEmail-btn {
```

Comment:
while this is all okay that you build custom CSS, and I will not stop you from doing it, there might be possibility I will throw this away, and replace with material-ui default btn